{"name": "Ошибка авторизации несуществующего курьера", "status": "passed", "description": "Проверка ошибки 404 при несуществующем курьере", "start": 1752323998152, "stop": 1752323998396, "uuid": "ec5ab906-05c0-4ab9-99c8-6aa0a6898e66", "historyId": "567cdf5b9ac7920ad3a7b27f2decce19", "testCaseId": "567cdf5b9ac7920ad3a7b27f2decce19", "fullName": "Tests.test_login_courier.TestLoginCourier#test_login_with_nonexistent_courier", "labels": [{"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-T2S04D8"}, {"name": "thread", "value": "80-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_login_courier"}]}